components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarPresenter.kt

```diff
@@ -69,10 +70,14 @@ class ToolbarPresenter(
         toolbar.setSearchTerms(session?.searchTerms ?: "")
         toolbar.displayProgress(session?.progress ?: 0)
         updateToolbarSecurity(session?.securityInfo ?: Session.SecurityInfo())
+        session?.let {
+            updateToolbarTrackingProtection(it)
+        }
```

Comment:
Changes here are dangerous since we are also landing #3783 which will move this from `browser-session` to `browser-state`. Let's sync who goes first.

Reply:
👍